install-tanzu-cli.md

```diff
@@ -97,9 +97,10 @@ and features.
 To install the Tanzu CLI and plug-ins:
 
 1. Sign in to [VMware Tanzu Network](https://network.tanzu.vmware.com).
-2. Go to the [TAP Release v1.1.0 > tanzu-cli-v0.11.2 download tile page](https://network.pivotal.io/products/tanzu-application-platform/#/releases/1078790/file_groups/7867).
-3. Download the Tanzu framework bundle for your operating system.
-4. (Optional) If an earlier upgrade attempt failed, it might be best to uninstall the previous version of the Tanzu CLI and associated plug-ins and files. To do so, see [Remove Tanzu CLI, plug-ins, and associated files](uninstall.html#remove-tanzu-cli).
+2. Go to the [TAP Product Page](https://network.pivotal.io/products/tanzu-application-platform).
+3. Select `Release 1.2.0` from the release dropdown menu (or the latest `...build` release for TAP 1.2.0)
```

Comment:
this is great for internal teams, but the `...build` part won't show up for customers. Not a big deal, but in a distant edge case I can imagine a customer saying "huh, I don't see anything with 'build' in it"